khal/aux.py

```diff
@@ -112,6 +112,8 @@ def calc_day(dayname):
         return today
     if dayname == 'tomorrow':
         return today + timedelta(days=1)
+    if dayname == 'now':
```

Comment:
Calc day actually returns date time objects, and used to call datetime.today() which is the same as datetime.now(), whereas date.today() is a date of today. I updated the code to have it return the start of today, and left now as an option there since the return type is the same, let me know your thoughts/if I should move now

> On Jun 6, 2016, at 5:49 AM, Christian Geier <notifications@github.com> wrote:
> 
> In khal/aux.py:
> 
> > @@ -112,6 +112,8 @@ def calc_day(dayname):
> >          return today
> >      if dayname == 'tomorrow':
> >          return today + timedelta(days=1)
> > +    if dayname == 'now':
> this function is called calc_day and currently only returns datetime.date objects, I don't think we should start returning datetime.datetime objects from here.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>